### PR TITLE
Add documentation articles 

### LINKED ISSUE #8: How-tos and development documentation
Hi, @Naylin15 

Change the name "Wiki" to "How-tos" and add these articles; and add the sub menu items to the Development section. If these pages does not exist, please, create an issue and request it. 👍 

**How-tos**

- [x] How to install
- [x] How to use
- [x] Use Cases
- [x] How it works
- [x] How to contribute
- [x] How to Translate

**Development**

- [x] Code documentation
- [x] Code coverage
- [x] Test report

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Add the documentation articles following the HackMD provided by @ingluife 

https://hackmd.io/cESB4-AARAWKI1JdGk0lUQ?both
https://hackmd.io/6FD4ag0STM-_a_ueP9941w?view

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@Naylin15 |26|

### References

Closes #8 
Related #3 
Depends on #N/A

### CODE REVIEW

**Review by ajsb85 — APPROVED** [review]

**Review by ajsb85 — APPROVED** [review]

**Review by ajsb85 — APPROVED** [review]

**Comment by Naylin15** [comment]
The documentation is finished, I'm having a small issue with the css for the tables, not taking the zebra style

captura de pantalla 2018-07-18 a la s 12 14 34 p m

**Review by ajsb85 — APPROVED** [review]

**Comment by Naylin15** [comment]
The issue with the zebra style is with browser Chrome in Firefox it displays the style correctly
captura de pantalla 2018-07-18 a la s 3 54 54 p m

**Review by ingluife — APPROVED** [review]

**Comment by Naylin15** [comment]
Issue style solved, it was caused by cache

File: _posts/static/2018-07-16-installation-pl.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: Installation
+permalink: howtos/installation-pl
+description: Building and configuring the environment
+category: plugin
+date: 2018-07-17
+---
+
+## Get the Docker Environment
+
+To start with the installation process, we must get the [Flyve MDM Docker Environment](https://github.com/flyve-mdm/docker-image) from its source.
```
Comment (ingluife) [inline _posts/static/2018-07-16-installation-pl.md]: Hi, @Naylin15. 

In this line, we must update the repository source to:

> https://github.com/flyve-mdm/docker-environment

File: _posts/static/2018-07-16-installation-pl.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- Using git, clone the repo: ```git clone https://github.com/flyve-mdm/docker-image.git```
```
Comment (ingluife) [inline _posts/static/2018-07-16-installation-pl.md]: The same here.

File: _posts/static/2018-07-16-installation-pl.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- [Download the project from GitHub](https://github.com/flyve-mdm/docker-image/archive/develop.zip)
+- [From GitHub Releases](https://github.com/flyve-mdm/docker-image/releases)
```
Comment (ingluife) [inline _posts/static/2018-07-16-installation-pl.md]: The same here.

**Review by ingluife — CHANGES_REQUESTED** [review]

**Review by ingluife — APPROVED** [review]

**Pull request merged**